engine/crates/engine/src/validation/mod.rs

```diff
@@ -32,6 +32,12 @@ pub struct ValidationResult {
 
     /// Query depth
     pub depth: usize,
+
+    /// Query height
```

Comment:
Not super obvious what `height` meant before reading the tests to be honest^^

Reply:
Added a comment.